modules/docs/arrow-docs/docs/docs/datatypes/nonemptylist/README.md

```diff
@@ -89,26 +89,23 @@ val nelOne: NonEmptyList<Int> = NonEmptyList.of(1)
 val nelTwo: NonEmptyList<Int> = NonEmptyList.of(2)
 val nelThree: NonEmptyList<Int> = NonEmptyList.of(3)
 
-ForNonEmptyList extensions {
-  binding {
-    val one = nelOne.bind()
-    val two = nelTwo.bind()
-    val three = nelThree.bind()
-    one + two + three
-  }.fix()
-}
+NonEmptyList.monad().binding {
```

Comment:
These bindings can just be top level and they should use the extension since it automatically downcasts the result type

Reply:
Yes, I realised it needs to be replaced with an import everywhere if we'd like for it to pass :D I'll get to it over the weekend.